client/e2e/protractor-ci.conf.js

```diff
@@ -1,9 +1,10 @@
 const config = require('./protractor.conf').config;
 
+config.chromeDriver = process.env.CHROMEWEBDRIVER;
```

Comment:
Since we aren't running e2e tests on github, we could technically remove this file and the CI configuration for e2e tests, but feel free to keep it if you want to continue trying to make it work and don't want to lose what has been written.

Reply:
Thanks. Doesn't protractor still need this file to run locally though?

Reply:
Removed it, thanks for explaining this to me in Slack.